packages/config-plugins/src/android/SplashScreen.ts

```diff
@@ -44,8 +44,11 @@ export function getSplashScreenConfig(config: ExpoConfig): AndroidSplashScreenCo
 }
 
 export async function setSplashScreenAsync(config: ExpoConfig, projectRoot: string) {
+  const majorVersionString = config.sdkVersion?.split('.').shift();
   const splashScreenIsSupported =
-    config.sdkVersion === '39.0.0' || config.sdkVersion === '40.0.0' || !config.sdkVersion;
+    (majorVersionString && Number(majorVersionString) >= 39) ||
```

Comment:
Would this error out on UNVERSIONED? (UNVERSIONED should always be used afaict).

Reply:
yeah that's a good point. `UNVERSIONED".split('.').shift()` -> `"UNVERSIONED"` - so `!majorVersionString` wouldn't handle this case